activestorage/lib/active_storage.rb

```diff
@@ -33,6 +33,11 @@ module ActiveStorage
 
   autoload :Attached
   autoload :Service
+  autoload :Previewer
 
   mattr_accessor :verifier
+
+  mattr_accessor :previewers do
+    [ ActiveStorage::Previewer::PdfPreviewer, ActiveStorage::Previewer::VideoPreviewer ]
```

Comment:
Why do we, and users who write their own previewer, have to add previewers here manually? Wouldn't an `inherited` hook work better?